notebooks/daniel_notebook.ipynb

```diff
@@ -1,191 +1,34 @@
 {
```

Comment:
<p><s>"As you can see,</s></p><p> </p><p>you can remove lead-ups in sentences. It makes them shorter/easier to read</p><p> </p><p><s>In order to simplify</s> &lt;- Add the forces together to find sum of forces. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>